src/pages/Hyun Yong Jeong/Login/Login.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
+import { useState } from 'react';
+
+const Login = () => {
+  const [userId, saveUserId] = useState('');
+  const [userPassword, saveUserPassword] = useState('');
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    navigate('/main');
+  };
+  const goSignup = () => {
+    navigate('/Signup');
+  };
+
+  const isInputValid = userId.includes('@') && userPassword.length >= 5;
+
+  return (
+    <>
```

Comment:
component를 생성할 때 지켜야 할 컨벤션이 혹시 기억나시나요??
아래 예시처럼 최상단의 태그는 컴포넌트의 이름과 같은className을 가질 수 있도록 해주세요!
해당 구조처럼 작성되어야 css코드를 작성할 때 원활한 nesting구조를 가져갈 수 있습니다
```suggestion
    <div className="login">
```